src/pages/Portfolio/LendingStats/BadgesSection.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-	<section data-testid="lend-stat-badges" class="stats-section">
-		<h2 class="tw-flex tw-gap-2 tw-mb-4">
+	<section data-testid="lend-stat-badges" class="stats-section tw-bg-gray-100 tw-py-4">
```

Comment:
When this is merged, please post in #b2c-eng-team for feedback from Nathan. The gray box looks a little out of place on the lending stats page.

Reply:
Agreed! But an specific requirement from Nathan...

> Gray BG really matters here?
> Yeah I think so because the cards will be white on white and the drop shadows are hard to see especially on desktop where there are a lot in a row!